crates/bevy_render/src/texture/image.rs

```diff
@@ -99,6 +99,26 @@ impl ImageFormat {
             ImageFormat::Basis | ImageFormat::Ktx2 => return None,
         })
     }
+
+    pub fn from_image_crate_format(format: image::ImageFormat) -> Option<Self> {
```

Comment:
```suggestion
    /// Attempts to convert from [`image::ImageFormat`].
    ///
    /// Unsupported formats will become [`None`].
    pub fn from_image_crate_format(format: image::ImageFormat) -> Option<Self> {
```

Reply:
I'd also be in favor updating the docs on the sibling methods to match in this PR.